experiment/practicaltest.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
There are no explanations on any of the answers. Please add them

Reply:
These were practical questions, and even after trying couldn't write enough theoretical explanations about options. All of it is numerical, and can be got only after running (or dry-run) of the given code.

Reply:
The idea about explanation is that you need to understand what mistake the user made when he reached a specific wrong answer and then point that out to the user so I don't agree with your explanation. Let's discuss this too on the call